packages/core/src/browser/progress-status-bar-item.ts

```diff
@@ -50,10 +57,8 @@ export class ProgressStatusBarItem implements ProgressClient {
         }
```

Comment:
> Should we delete it when progress is done?

@akosyakov, please add `this.messagesByProgress.delete(progressId)` in the `else` branch 🙏